### PR TITLE
Logger Modification

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### What does this implement/fix? Explain your changes.
- `logger` function modified

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR 💯

File: pyrgg/test.py [hunk 1]
```diff
@@ -31,8 +31,6 @@
 True
 >>> is_float(None)
 False
->>> logger(2,2,2,2,2,2,2,2,2,2,2,2)
-[Error] Logger Failed!
```
Comment (sadrasabouri) [inline pyrgg/test.py:35]: It'll be better if we add a test which make a failure here. (According to Codecov test failure)

**Review by sadrasabouri — COMMENTED** [review]
That was nice. I left a comment hope it'll be useful.

File: pyrgg/test.py (hunk 1 above)
Comment (sepandhaghighi) [reply pyrgg/test.py:35]: You are right, but I couldn't find an example.
Do you have any idea?

Comment (sadrasabouri) [reply pyrgg/test.py:35]: No.
Just file opening exceptions seems to be a try-except block trigger here.

**Pull request merged**